### DISCUSSION TITLE
Latest Dev build (22526) - Explorer won't start with ExplorerPatcher installed

### DISCUSSION THREAD

**Discussion by Andre-10** [valinet/ExplorerPatcher#634] [General]
I know it's only Windows 11 Dev build, but could be a warning that MS is taking steps to remove old Taskbar from Windows 11 code, or just preventing EP to run.

After upgrade to a new build just black screen. Had to start Task manager (Ctrl+Alt+Del) and run cmd - rename EP to EP_uninstall.exe and run it.

Repeated EP install to be sure that it's EP, and black screen again

**Comment by valinet** [comment]
Yeah, well, tbh, I don’t have time for that. They do whatever they want, fuck it however they want, eventually a new stable version will be released, if still necessary, I will then look into what’s there anymore and try to fix. Not upgrading is a possible solution, as well as maybe using the explorer.exe from 22000. Maybe enable the console, check to see what it says, or run explorer.exe under a debugger and see where it stops, to find out more details about what really changed. I think they may have slightly tweaked some internal thing, idk, it’s typical of them. Idk, also, check if UndockingDisabled still works, that’s an indicator of whether the old taskbar is still there, or running explorer under SYSTEM account. Supporting this thing takes a lot anyway, keeping up to date with Windows Insider is honestly a waste of brain power, it eventually teaches you nothing, it’s tedious, boring and it would take up all my time which is stupid, as the fight is fruitless. It’s their code, if they want to screw it up for good, they will do it eventually. I’d be more interested in developing a proper alternative shell, that’s one of my long time plans, EP was mostly meant as a way of learning and as a bandaid until something more proper and not dependent on a third party’s idiotic actions is developed.

Interesting report though, thanks.

Or I might just go back to GNU/Linux, like, the situation here is getting worse by the day. Pluton chip being the latest stunt.

**Reply by Andre-10** [reply]
Yeah, I know. I've reported it as a warning. It would really be insane to keep up with insider builds...

Thanks for answering

**Reply by valinet** [reply]
No worries. We’ll see. As I said, maybe I have some time one day or someone else picks it up and sees what this is all about. Maybe it’s not something alarming.

**Comment by ghost** [comment]
I just installed the ExplorerPatcher. Nice work! I now have my Quick Launch Area next to my Start Button.

I have a question about the height of the Taskbar.  Can it be 1/2 it's current height?  I used W10 with small icons in the Quick Launch Area and I could no longer see the date. I really don't care about seeing the date, I can always hover my cursor over the time and it shows me the date in a balloon.

Thanks again for this wonderful tool.

**Reply by valinet** [reply]
Right click taskbar Properties, Taskbar icon size.

**Reply by ghost** [reply]
Outstanding!  Thank you so much!

**Reply by ghost** [reply]
> Right click taskbar Properties, Taskbar icon size.

I forgot to mention that I also use Start11 from stardock.com. There are no conflicts while using both apps. Again - kudos! If anyone ever used QLA all these years it's thrilling to have it back due to the ExplorerPatcher. Shortcuts to files, folders, programs. Just wonderful! 

![startmenuandquicklauncharea](https://user-images.githubusercontent.com/34069523/148594054-c7647869-6cf9-48fc-942b-786e7af01310.jpg)

**Comment by valinet** [comment]
Well, I cannot seem to reproduce this...

![image](https://user-images.githubusercontent.com/6503598/148506962-bb783397-e7e0-445b-a213-05c9f879baf9.png)

Anyway, at least the old taskbar is still there.

**Reply by Andre-10** [reply]
Latest version .3 is working fine.. Symbols downloaded, and Taskbar and desktop are here again 

![image](https://user-images.githubusercontent.com/93877359/148582402-8448219d-c85b-4a5c-869e-4efcf4daf5a8.png)


Thanks @valinet

**Comment by radische** [comment]
Sam problem as Andre ... Build 22526 downloaded and installed ... boots to black screen with cursor. Tried several ways to repair and everything failed. It looked like I had bricked my Surface Pro 6, but was eventually able to get to a "fix-it" page by shutting down computer and simultaneously depressing the power and the volume down buttons until Windows icon appeared. Fix-it didn't work but allowed me to troubleshoot where I  was able to "reset machine and got it working again in Build 22526.  I then tried to download the ep setup file again, but Microsoft tried to warn me that the file was unsafe. Even so, I choose to ignore the warning and continued the download, but Microsoft deleted the downloaded file without my permission. I then tried to use the machine reset to 22526, but had major problems with the Build, so I have now gone back and am in the process of installing 22000.376 to see if that (now) has problems with the EP installation. I'll report on that below. 

I do have ExplorerPatcher on 6 other machines at my home which are running the standard version of Windows 11 successfully. 

Ah, I now have 22000.376 on the Surface Pro 6. I am attempting to download the ep setup file. "ep_setup.exe isn't commonly downloaded. Make sure you trust ep_setup.exe before you open it." I click on the 3 dots above. I click on "Keep". Another warning appears and I click on "Show more". I click on "Keep Anyway". I go to file and execute it ... ExplorerPatcher is running. Go to "Properties" on task bar and make my selections. Everything works with 22000.376, even though Microsoft tries to stop the download. 

Hooray. Now I have to install all my other software!

**Reply by Andre-10** [reply]
radishe, as _ wrote above.. in the case like this, pres Ctrl+Alt+Del and start Task manager. In the run box (Menu - File) type cmd, and you'll get command prompt. Navigate to the folder you have ep_setup.exe and uninstall it. _

And system backups are essential when you play with Windows insider builds.

**Comment by nemethviktor** [comment]
If it's of any help, I've reinstalled an older version of EP on this specific Insider build (348.39.2) and that works fine without issues.
